.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
+python:
+  - "2.7"
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install mpi-default-bin mpi-default-dev
+
+  # LAMMPS
+  #  - build lammps as shared library
+  #  - and install LAMMPS python wrapper
+  - git clone git://git.lammps.org/lammps-ro.git mylammps
+  # checkout a recent stable version (from 28 June 2014)
+  - cd mylammps/src && git checkout r12164
+  # make shared library for python
+  - make makeshlib && make -f Makefile.shlib ubuntu_simple
+  - cd ../python && sudo python install.py /usr/lib/ $VIRTUAL_ENV/lib/python2.7/site-packages/
+  # make lammps executable and add to path
+  - cd ../src
+  - make ubuntu_simple
+  - ln -s lmp_ubuntu_simple lammps
+  - export PATH=$TRAVIS_BUILD_DIR/mylammps/src/:$PATH
+  - cd $TRAVIS_BUILD_DIR
+
+  # use current master of simphony
+  - pip install git+git://github.com/simphony/simphony-common.git#egg=simphony
```

Comment:
Having a `DataContainer` whose value has a variable length is going to make serialization extremely difficult. If I understand the information in the PAIR_POTENTIALS is global only needed for the configuration and not for the Particles themselves

I would suggest the following:
- We should split the CUBA enums into an enum for `DataContainer` and one for `ConfigContainer`.
- Create a new type of `DataContainer` (e.g `ConfigContainer`) that will support the necessary keywords. 

Please note that if PAIR_POTENTIAL is not something universal e.g. used by all engines then it is probably better to have a special configuration object with attributes instead of a dictionary like container.